packages/web-components/fast-components/src/skeleton/fixtures/base.html

```diff
@@ -0,0 +1,169 @@
+                margin-top: 10px;
+                height: 10px;
+            "
+            shape="rect"
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                margin-top: 10px;
+                height: 10px;
+            "
+            shape="rect"
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                margin-top: 10px;
+                height: 10px;
+            "
+            shape="rect"
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                width: 75px;
+                height: 30px;
+                margin-top: 20px;
+                margin-bottom: 10px;
+            "
+            shape="rect"
+        ></fluent-skeleton>
+    </div>
+
+    <h4>Used as element blocks with shimmer effect element</h4>
+    <div class="card-placeholder">
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                width: 50px;
+                height: 50px;
+            "
+            shape="circle"
+            shimmer
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                margin-top: 10px;
+                height: 10px;
+            "
+            shape="rect"
+            shimmer
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                margin-top: 10px;
+                height: 10px;
+            "
+            shape="rect"
+            shimmer
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                margin-top: 10px;
+                height: 10px;
+            "
+            shape="rect"
+            shimmer
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                width: 75px;
+                height: 30px;
+                margin-top: 20px;
+                margin-bottom: 10px;
+            "
+            shape="rect"
+            shimmer
+        ></fluent-skeleton>
+    </div>
+
+    <h4>Using SVG via Pattern attribute</h4>
+    <fluent-skeleton
+        style="
+            --neutral-fill-rest: #e1dfdd;
+            border-radius: 4px;
+            width: 500px;
+            height: 250px;
+        "
+        shape="rect"
+        pattern="https://static.fast.design/assets/skeleton-test-pattern.svg"
```

Comment:
Referencing this soon-to-be-CDN-hosted SVG means the fixture relies on an internet connection to properly execute. Can we include the file next to this fixture file and use a relative URL here?

Reply:
Since I'm using the `<object>` element I can set the fallback image source to a local file, but including the file next to the fixture doesn't work. @chrisdholt thoughts?

Reply:
@radium-v would you be open to us taking this as a follow-up issue? Agree on it being an issue for long-term dev and the ability to do so without an internet connection. I'd prefer to not block on this and figure out a good method for testing with images. While I'm hesitant to do so, I think we're going to need to and I'd like to have a solid solution in place.

What do you think?

Reply:
I'll create a ticket to follow up on this.